opp-distribution-service/contracts/testnet/sepolia-to-boba/scripts/testContract.js

```diff
@@ -0,0 +1,21 @@
+require('dotenv');
+
+const { ethers } = require("hardhat");
+const { SEPOLIA_CONTRACT_ADDRESS } = process.env;
+
+async function main() {
+    const contractAddress = SEPOLIA_CONTRACT_ADDRESS;
+    const Contract = await ethers.getContractFactory("SepoliaToBobaBridge");
+    const contract = Contract.attach(contractAddress);
+
+    // call testResponse func
+    const response = await contract.testResponse();
+    console.log("Response from contract: ", response);
```

Comment:
Response will come from this line in the contract: https://github.com/opportunity-hack/ohack-currency-prototype/pull/2/files#diff-c54ca2d22b04b7c1f4b86fb00297a67d5523aee2ad7e4873f082d7ac2dc4fac7R43